hil/api.py

```diff
@@ -1084,14 +1090,46 @@ def port_revert(switch, port):
 
     if port.nic is None:
         raise errors.NotFoundError(port.label + " not attached")
-    if port.nic.current_action:
-        raise errors.BlockedError("Port already has a pending action.")
+    check_pending_action(port.nic)
+    unique_id = str(uuid.uuid4())
 
-    db.session.add(model.NetworkingAction(type='revert_port',
-                                          nic=port.nic,
-                                          channel='',
-                                          new_network=None))
+    action = model.NetworkingAction(type='revert_port',
+                                    nic=port.nic,
+                                    channel='',
+                                    uuid=unique_id,
+                                    status='PENDING',
+                                    new_network=None)
+
+    db.session.add(action)
     db.session.commit()
+    return json.dumps({'status_id': unique_id})
+
+
+@rest_call('GET', '/networking_action/<status_id>', Schema({
+    'status_id': basestring}))
+def show_networking_action(status_id):
+    """Returns the status of the networking action by finding the status_id
+    in the networking actions table.
+    """
+    action = model.NetworkingAction.query.filter_by(uuid=status_id).first()
+    if action is None:
```

Comment:
`if not action`? 

Just curious, what would be the best approach for handling vars those have value of `None`

Reply:
I think that would work too, I just thought `if action is none` reads like a real sentence.

Reply:
I just happen to think of that, I have no idea which approach is more pythonic. Personally, I use both approaches

Reply:
The existing precedent in the codebase is to do `if ... is None`. We tend to shy away from using things that aren't stictly booleans as conditions. That's certainly not universal in the Python world though.